src/main/java/com/mixam/mxjdf4/sdk/MxJdf4.java

```diff
@@ -85,9 +85,6 @@ public void setJob(Job job) {
     }
 
     public Export getExport() {
-        if (export == null) {
-            export = new Export();
-        }
```

Comment:
## ATTENTION EFFIE

@Effie -- Just like the `shipment` element, it seems to me that Mxjdf documents for ripped proof orders should not contain an `export` element.  I searched the code in `mixam-shop`, and I can find nothing that will break if these 3 lines are removed.

Reply:
Also price & currency.